Model/Source/ProductList.php

```diff
@@ -26,33 +19,36 @@ class ProductList
     const CROSSSELL = 'crosssell';
     const WIDGET = 'widget';
 
+    /**
+     * @return array
+     */
     public function toOptionArray()
     {
-        return array(
-            array(
+        return [
+            [
                 'value' => '',
-                'label' => __('None')
-            ),
-            array(
+                'label' => __('None'),
+            ],
+            [
                 'value' => self::CATEGORY,
-                'label' => __('Category and Search Result Pages')
-            ),
-            array(
+                'label' => __('Category and Search Result Pages'),
+            ],
+            [
                 'value' => self::UPSELL,
-                'label' => __('Upsells on Product Pages')
-            ),
-            array(
+                'label' => __('Upsells on Product Pages'),
```

Comment:
In Magento, they use the name format of "Up-Sell Products" (dash)

Reply:
Fixed.